default/col_buildings.txt

```diff
@@ -184,7 +184,7 @@ BuildingType
             activation = Turn low = Source.CreationTurn + 2
             effects = Destroy
     ]
-    icon = "icons/building/generic_building.png"
+    icon = "icons/species/abaddonnian.png"
```

Comment:
IMHO building icons should be in buildings folder. How about to copy them to buildings and rename?